quotefault/models.py

```diff
@@ -0,0 +1,90 @@
+"""
+Defines the application's database models
+"""
+import binascii
+from datetime import datetime
+import os
+from sqlalchemy import UniqueConstraint
+from quotefault import db
+
+# create the quote table with all relevant columns
+class Quote(db.Model):
+    """
+    Quote table in SQL
+    """
+    __tablename__ = 'quote'
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    submitter = db.Column(db.String(80), nullable=False)
+    quote = db.Column(db.String(200), unique=True, nullable=False)
+    speaker = db.Column(db.String(50), nullable=False)
+    quote_time = db.Column(db.DateTime, nullable=False)
+    hidden = db.Column(db.Boolean, default=False)
+
+    votes = db.relationship('Vote', back_populates='quote')
+    reports = db.relationship('Report', back_populates='quote')
+
+    # initialize a row for the Quote table
+    def __init__(self, submitter, quote, speaker):
+        self.quote_time = datetime.now()
+        self.submitter = submitter
+        self.quote = quote
+        self.speaker = speaker
+
+
```

Comment:
Newline >:(

Reply:
I'm fine with double newlines between classes

Reply:
Actually, I made these double newlines, so I'm going to mark this resolved